### ISSUE TITLE
Discord community chat

### ISSUE THREAD

**Issue by odlainepre** [tradingview/lightweight-charts#241]
Hi! 

I did not find on the Internet a chat of a community of developers who would use Tradingview solutions (private `charting_library` and public - `lightweight-charts`). I use both solutions, and in the early stages there were a lot of typical questions. 

I can create and maintain a Telegram channel ex: `@tradingview_dev` community chat for private and public users, for answers basic questions. It will be great if you can update the readme.me and add a link.

What do u think about?

**Comment by timocov** [comment]
That's a good idea! We've thought about creating chat in gitter but still didn't do it properly due some reasons.

What do you think about [Discord](https://discordapp.com)? It looks like [they are open to open-source communities](https://discordapp.com/open-source) (yeah, charting_library is closed source actually), and some communities moved to it (I known that there is TypeScript, VueJS and React communities; I'm not sure how much they are popular though).

Telegram is awesome, but it had (I'm not sure about right now) issues with transferring ownership of the group between users, what might be a problem in the future.

Also, in Descord it might be different channels about specific product (like lightweight-charts, charting library, trading terminal), in Telegram you need to have either separate groups for each product or mix all products in one group.

**Comment by nikitamarcius** [comment]
**Discord is a great solution**, which is very popular outside of Russia. It will completely solve the problem of different channels (TypeScript, VueJS and React) and rights (admin, staff, moderator, etc). 

If you take a Russian case with a Telegram, then there will be mix of all. There is less freedom in Telegram on the issue of customization of rights (admin, user).

I agree, the Discord will be better with different channels and separation of rights.

> We've thought about creating chat in gitter

No gitter, its bad startup . Poor interface, search and everything is indexed by search engine.


If you agree to the Discord, I am ready run as it should.

**Comment by timocov** [comment]
> No gitter, its bad startup . Poor interface, search and everything is indexed by search engine.

Yep, agree :) The only thing is good in gitter is linking with github, but for chatting it might be not so much needed.

> If you agree to the Discord, I am ready run as it should.

https://discord.gg/26AEtb please tell your nickname and I'll grant you admin access (or you can create it yourself and put the link here and I remove that server).

**Comment by nikitamarcius** [comment]
Discord: Nikita Marcius#2274

**Comment by timocov** [comment]
Done https://github.com/tradingview/lightweight-charts/commit/ff76b79f8bcb6ad33001afb9fe5e3aeecc278800. Many thanks @nikitamarcius!

Invite link https://discord.gg/E6UthXZ.
